src/metabase/models/permissions.clj

```diff
@@ -487,12 +487,24 @@
   permissions path for the table. Otherwise, defaults to the root path ('/'), thus restricting writes to admins."
   [db-id schema table-id]
   (let [f (u/ignore-exceptions
-           (classloader/require ' metabase-enterprise.advanced-permissions.models.permissions)
-           (resolve ' metabase-enterprise.advanced-permissions.models.permissions/data-model-write-perms-path))]
-    (if (and f premium-features/enable-advanced-permissions?)
+           (classloader/require 'metabase-enterprise.advanced-permissions.models.permissions)
+           (resolve 'metabase-enterprise.advanced-permissions.models.permissions/data-model-write-perms-path))]
+    (if (and f (premium-features/enable-advanced-permissions?))
       (f db-id schema table-id)
       "/")))
 
+(s/defn db-details-write-perms-path :- Path
```

Comment:
This function is basically the same as the one above for data model perms, but I'm going to experiment using a macro to abstract this stuff away in a future PR. (see [this thread](https://metaboat.slack.com/archives/CKZEMT1MJ/p1649176488140789) for context)